### PR TITLE
feat(auth-pages): added signin and signup pages

### PR DESCRIPTION
**Pull request by pakyardim**
This commit adds the SignIn and SignUp pages to the app, along with the necessary styling and UI components. The SignIn page includes a login form with email and password fields, as well as a "Forgot password" link. The SignUp page includes a registration form with fields for name, username, email, password, and birthday. Both pages use a consistent color scheme and typography to maintain a cohesive look and feel with the rest of the app. This feature will provide users with a streamlined and user-friendly way to sign in and sign up for the app.

### CODE REVIEW

File: package.json
Comment (kaganecee) [inline package.json:1]: Are all the libraries you added being used? If it's not can you delete related dependencies. It seems you don't used some of them in this commit.

File: src/App.scss
Comment (kaganecee) [inline src/App.scss:1]: Can you move related css to assets/index.scss file. And please remove unnecessary classes.

File: src/components/UI/AuthSection.js
Comment (kaganecee) [inline src/components/UI/AuthSection.js:1]: Can you move this constants to constant file which is you need to create new one

**Review by kaganecee — CHANGES_REQUESTED** [review]

**Comment by pakyardim** [comment]
New commits are added by covering the changes requested

**Pull request merged**